### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by maurogestoso**
I modified the README to be an introduction to the workshop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,69 @@
...
+
+### Opción B: Local
+
+​Para este proyecto necesitarás [Git](https://git-scm.com/downloads), [Node y NPM](https://nodejs.org) instalados en tu entorno local y tu editor de código favorito. Nosotres utilizamos [Visual Studio Code](https://code.visualstudio.com/).
+​
+Una vez hayas hecho un _fork_ de este repositorio, puedes clonarlo a tu ordenador con el siguiente comando:
+​
+
+```bash
+git clone URL git@github.com:mercadona/mo.workshop.tdd-react.git
```
Comment (alextorres94) [inline README.md:50]:
```suggestion
git clone git@github.com:mercadona/mo.workshop.tdd-react.git
```

File: README.md [hunk 2]
```diff
@@ -1,20 +1,69 @@
-# Getting Started with TDD React Workshop
+# Taller React TDD
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Introducción
 
-## Available Scripts
+Hola 👋🏼 Somos el equipo de Frontend de Mercadona Tech. Nuestra intención con este taller es mostrar nuestra forma de trabajar para compartir conocimiento con la comunidad y poner nuestro granito de arena para mejorar la diversidad en nuestro sector.
```
Comment (alextorres94) [inline README.md:5]: Neutral emoji 😄 

```suggestion
Hola 👋 Somos el equipo de Frontend de Mercadona Tech. Nuestra intención con este taller es mostrar nuestra forma de trabajar para compartir conocimiento con la comunidad y poner nuestro granito de arena para mejorar la diversidad en nuestro sector.
```

File: README.md [hunk 3]
```diff
@@ -1,20 +1,69 @@
...
+- Cómo iterar un producto añadiendo funcionalidad de forma progresiva
+- Cómo desarrollar un producto desde el punto de viste del usuario
+- Cómo colaborar de forma efectiva haciendo _pair programming_
+  ​
+  Aunque todos estos principios se puede aplicar a cualquier tecnología, nos centraremos en las tecnologías que utilizamos para desarrollar la web de Mercadona Online:
+  ​
+- HTML + CSS
+- JavaScript
+- React
+- Jest + Testing Library
```
Comment (alextorres94) [inline README.md:22]:
```suggestion
- Cómo escribir tests que te den confianza de que tu código funciona.
- Cómo iterar un producto añadiendo funcionalidad de forma progresiva.
- Cómo desarrollar un producto desde el punto de viste del usuario.
- Cómo colaborar de forma efectiva haciendo _pair programming_.
​
Aunque todos estos principios se puede aplicar a cualquier tecnología, nos centraremos en las tecnologías que utilizamos para desarrollar la web de Mercadona Online:
  ​
- HTML + CSS
- JavaScript
- React
- Jest + Testing Library
```

**Review by juandiegombr — APPROVED** [review]
Great! I would add the steps to start working on the project. Run the tests with `npm run test` and remove the `fit` one by one.

**Pull request merged**